### PR TITLE
Introduce HipChat notifications

### PR DESCRIPTION
**Pull request by maxinspace**
Repo: https://github.com/maximlarionov/feedback-form

### CODE REVIEW

File: feedback-form/README.md [hunk 1]
```diff
@@ -162,3 +266,103 @@ feature "Create Feedback." do
...
+end
+```
+
+```ruby
+# Gemfile.rb
+
+gem 'hipchat'
+
+group :development, :test do
+  gem 'rspec-its'
```
Comment (maxinspace) [inline feedback-form/README.md:316]: actually was adding this gem every single time I work with rails-base.

File: feedback-form/README.md [hunk 2]
```diff
@@ -4,19 +4,28 @@
 # app/controllers/feedbacks_controller.rb
 class FeedbacksController < ApplicationController
   expose(:feedback) { Feedback.new(feedback_attributes) }
+  expose(:decorated_feedback) { FeedbackDecorator.new(feedback) }
```
Comment (timurvafin) [inline feedback-form/README.md]: Please use `expose_decorated`

File: feedback-form/README.md [hunk 3]
```diff
@@ -4,19 +4,28 @@
...
 
   def create
-    ApplicationMailer.feedback(feedback).deliver_now! if feedback.save
+    perform_delivery if feedback.valid?
     respond_with(feedback, location: root_path)
   end
 
   private
 
+  def perform_delivery
```
Comment (timurvafin) [inline feedback-form/README.md]: Why don't extract this method as interactor?

File: feedback-form/README.md [hunk 4]
```diff
@@ -30,47 +39,103 @@ class Feedback
...
+    organize HipchatInteractor::Connector, HipchatInteractor::Messenger
+  end
+end
+```
 
-      .form-actions
-        = f.button :submit, "Submit"
+```ruby
+# app/interactors/hipchat_interactor/connector.rb
+module HipchatInteractor
```
Comment (timurvafin) [inline feedback-form/README.md]: I don't see any reason to use `Connector` as Interactor. Intercators should incapsulate business logic.
Better create separate class to wrap `HipChat::Client`

File: feedback-form/README.md [hunk 5]
```diff
@@ -162,3 +266,103 @@ feature "Create Feedback." do
...
 end
 ```
+
+```ruby
+# spec/interactors/hipchat_interactor/organizer_spec.rb
+
+require "rails_helper"
+
+describe HipchatInteractor::Organizer do
+  let(:params) do
```
Comment (timurvafin) [inline feedback-form/README.md]: Could we use Factory here?

**Comment by maxinspace** [comment]
thanks @timurvafin , fixed those points.

**Pull request closed without merging**